Rest content type docs (#1195)
microsoft/cadl: packages/website/src/docs/standard-library/rest/operations.md

}
```

## Content type

### Defaults

Depending on the body of the operation http library will assume different content types:

- `bytes`: `application/octet-stream`
- `string`: `text/plain`
- an `object` or anything else: `application/json`

Examples:

```cadl
op download(): bytes; // response content type is application/octet-stream
op upload(@body file: bytes): void; // request content type is application/octet-stream
op getContent(): string; // response content type is text/plain
op getPet(): {
  // response content type is application/json
  name: string;
};
```

### Specify content type

> The content type for an operation can be specified using specifying an header property called `contentType`

Reviewer: ```suggestion
The content type for an operation can be specified by including a header parameter named `contentType`.
```
